src/state/megaStore.tsx

```diff
@@ -272,7 +279,12 @@ export const Provider: ParentComponent = (props) => {
                             balance: newBalance,
                             last_sync: Date.now(),
                             price: 1,
-                            fiat: FIAT_OPTIONS[BTC_INDEX]
+                            fiat: {
+                                label: "Bitcoin BTC",
+                                value: "BTC",
+                                hasSymbol: "₿",
+                                maxFractionalDigits: 8
+                            }
```

Comment:
How about we make this a `const` in currencies.ts and you cant use it here and put it in the array. That way we get the best of both worlds, safety and no duplication

Reply:
Went with a concat option with USD+BTC thus they are separate from the rest of the `FIAT_OPTIONS` and I put them at the front so that they are the most easily accessible options fro the user